File: Practicas/Marco-Oregel/Practica7/Ironman.py
from MarvelCharacter import MarvelCharacter
import random
class Ironman(MarvelCharacter):
    def __init__(self, name, alias, universe, age):
        super().__init__(name, alias, universe, age)

    
    def showPower(self):
        powers = ["JARVIS, Engage heads-up display!","Power up the suit!","I am Iron Man.", "Jarvis, drop my needle"]
        power = random.choice(powers)
        print(power)

    def getAge(self):
        return super().getAge()
    
    def updateAge(self, newAge):
        return super().updateAge(newAge)
       

File: Practicas/Yahir-Hermosillo/Parcial3/Practica7/IronMan.java
public class IronMan extends MarvelCharacter {
    public IronMan(String name, String alias, String universe, int age) {
        super(name, alias, universe, age);
    }

    @Override
    public void showPower() {
        System.out.println("Tengo mi armadura de nanotecnología");
    }
}